content/docs/gke/deploy/monitor-iap-setup.md

```diff
@@ -180,19 +113,8 @@ problems:
     If the backend is unhealthy, check the pods in `istio-system`:
     * `kubectl get pods -n istio-system`
     * The `istio-ingressgateway-XX` pods should be running
-    * Check the logs of `backend-updater-0`, `ingress-bootstrap-XX`, `iap-enabler-XX` to see if there is any error
-
-1. Now that the certificate exists, the Ingress resource should report that it 
-  is serving on HTTPS:
-
-    ```
-    kubectl -n istio-system get ingress
-    NAME            HOSTS                                                        ADDRESS          PORTS     AGE
-    envoy-ingress   mykubeflow.endpoints.myproject.cloud.goog   35.244.132.159   80, 443   1d
-    ```
-
-    If you don't see port 443, look at the Ingress events using 
-    `kubectl describe` to see if there are any errors.
+    * Check the logs of pod `backend-updater-0`, `iap-enabler-XX` to see if there is any error
```

Comment:
I think per @sarahmaddox's recommendations this should be reworded to address the reader.
e.g

* You can can the logs of pod ....

IIRC the sentence fails the zombie test. I.e. "Zombies check the logs of the pod"  would still be a proper english but not what I want.

Reply:
Actually, the usage is OK here. :) Telling people to do something ("check the logs...") is fine.

**More explanation of the zombies thing:**

The zombie test is to pick up passive voice, where it's unclear who should be doing something. For example, if this sentence said:

"The logs of pod `backend-updater-0`, `iap-enabler-XX` should be checked."

Then you can add "by zombies":

"The logs of pod `backend-updater-0`, `iap-enabler-XX` should be checked by zombies."

So, it's not clear who should do the checking.

Reply:
Thanks @sarahmaddox

Reply:
thanks